File: Solucao.UI/src/app/shared/models/calendar.ts
import { CalendarSpecifications } from "./calendarSpecifications";

export interface Calendar {
    id: string;
    equipament: string;
    equipamentId: string;
    lessee: string,
    clientId: string,
    date: string,
    note: string,
    time: string,
    technique: string,
    techniqueId: string,
    driver: string,
    driverId: string,
    status: string,
    startTime1: string,
    startTime: string,
    endTime1: string,
    endTime: string,
    calendarSpecifications: CalendarSpecifications[]
  }

File: Solucao.Application/Data/Entities/Calendar.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Solucao.Application.Data.Entities
{
    public class Calendar
    {
        public Guid Id { get; set; }
        public Guid? ParentId { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public Guid EquipamentId { get; set; }
        public string Note { get; set; }
        public Guid ClientId { get; set; }
        public string Status { get; set; }
        public bool Active { get; set; }
        public bool NoCadastre { get; set; }
        public string TemporaryName { get; set; }
        public DateTime Date { get; set; }
        public DateTime? StartTime { get; set; }
        public DateTime? EndTime { get; set; }
        public Guid? DriverId { get; set; }
        public Guid? TechniqueId { get; set; }
        public Guid UserId { get; set; }
        public int TravelOn { get; set; }
        public User User { get; set; }
        public Person Technique { get; set; }
        public Person Driver { get; set; }
        public Client Client { get; set; }
        public Equipament Equipament { get; set; }
        public IList<CalendarSpecifications> CalendarSpecifications { get; set; }
    }
}
